io/plugins/eu.esdihumboldt.hale.io.jdbc.msaccess/src/eu/esdihumboldt/hale/io/jdbc/msaccess/MsAccessJdbcIOSupplier.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2015 Data Harmonisation Panel
```

Comment:
We need to change the default (c) header. The (c) lies with wetransform for all new files.